getting_started/loadable_modules/lmdk_user_guide.rst

```diff
@@ -0,0 +1,25 @@
+.. _lmdk_user_guide:
+
+Loadable modules build guide using LMDK
+#######################################
+
+What is LMDK
+************
+
+LMDK(Loadable Module Development Kit) is a standalone package required to build loadable module. It is independent from SOF FW but contains necessary to interact with it.
+
+How to build
+************
+
+To build example loadable library execute:
+.. code-block:: bash 
+
+    $ cd libraries/example
+    $ mkdir build
+    $ cd build
+
+    $ cmake -DRIMAGE_COMMAND="/path/to/rimage" -DSIGNING_KEY="/path/to/signing/key.pem" ..
+    $ cmake --build .
```

Comment:
isn't there a shared folder or something with the headers? I don't quite see how the module can be build in a stand-alone manner and yet follow APIs.

Reply:
It is something similar to what was done with IADK https://github.com/thesofproject/converged-sof-modules

Reply:
right, but I see a set of threads on the need to change the module interface, so the odds of that module API to remain fixed are very slim IMHO...

Reply:
During designing native loadable modules it was taken to equation. If module interface grow It would be catch by CI. It will took some work but not that much.

Reply:
@plbossart I agree that in the long term we should refactor our SOF base FW code and sort out shared Module API and System Services headers that will be consumed for loadable modules building. I recommend to submit an issue/enhancement to track that activity. 

For now the CI check should be sufficient and while the Module API will evolve we should put pressure to expand it without breaking backwards compatibility.

Reply:
That's fine with me @mwasko @pjdobrowolski , change is the only constant in our line of work :-)

May I ask though that this documentation mentions possible changes as we mature the module API. We've got to set the expectation that changes are already anticipated so that partners/users are not surprised. Thanks!